app/views/devise/registrations/edit.html.haml

```diff
@@ -0,0 +1,34 @@
+%h2= t('.edit_resource', resource: resource_name.to_s.humanize)
+
+= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }, builder: GOVUKDesignSystemFormBuilder::FormBuilder) do |f|
```

Comment:
could move to `form_with` too. Would need testing to be sure

e.g. [example gist](https://gist.github.com/ParamagicDev/b264e64dec6a36770f74fe193cd8fd8e)

```
= form_with(model: resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }, builder: GOVUKDesignSystemFormBuilder::FormBuilder, local: true) do |f|
```

Reply:
Not sure if we need `html: { method: :post }`. It may just be `method: :post`, rather than nested.

[referenence](https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/FormHelper.html#method-i-form_with)